test/metabase/test_runner/assert_exprs/approximately_equal_test.clj

```diff
@@ -0,0 +1,50 @@
+(ns metabase.test-runner.assert-exprs.approximately-equal-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.test-runner.assert-exprs :as assert-exprs]
+   [metabase.util.date-2 :as u.date]))
+
+(comment assert-exprs/keep-me)
+
+(deftest ^:parallel passing-tests
+  (testing "basic equality"
+    (is (≈ 100 100)))
+  (testing "predicate function"
+    (is (≈ int? 100)))
```

Comment:
Seems like this could be `(int? 100)` instead, right?

Reply:
Makes more sense now that @bshepherdson explained that this could be applied as a leaf value, in something like

```
(is (≈ {:a int? :b even?} {:a 1 :b 2}))
```